src/shared/util.js

```diff
@@ -153,8 +153,7 @@ export function hasOwn (obj: Object | Array<*>, key: string): boolean {
 export function cached<F: Function> (fn: F): F {
   const cache = Object.create(null)
   return (function cachedFn (str: string) {
-    const hit = cache[str]
-    return hit || (cache[str] = fn(str))
+    return cache[str] || (cache[str] = fn(str))
```

Comment:
This is not really perf but rather code style. [terser](https://github.com/terser-js/terser) rewrites it to your version anyway.

Reply:
you are right, not familiar with commit convention.  seams can't edit pr title